Source/Controllers/MainViewControllers/SPDatabaseDocument.m

```diff
@@ -1420,7 +1420,7 @@ - (void) endTask
 		if (queryExecutionTimer) {
 			queryStartDate = [[NSDate alloc] init];
 			[self showQueryExecutionTime];
-			[queryExecutionTimer invalidate], SPClear(queryExecutionTimer);
```

Comment:
Did you mean to drop the `SPClear` here?

Reply:
Yes, as you can see one row below, it's there twice. :)